test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/smoke/ExternalMavenRepoIntegrationTest.java

```diff
@@ -118,30 +141,6 @@ public void testProcessFromExternalMavenRepo() {
         assertThat(outcome).hasSize(2);
         assertThat(outcome.get(0)).startsWith(HELLO_RULE);
         assertThat(outcome.get(1)).startsWith(WORLD_RULE);
-
-        try {
-            startAndCompleteUsertaskProcess();
-        } catch (KieServicesException ex) {
-            // OK for drools test.
-            assertThat(ex.getMessage()).contains("Server that this client is connected to has no capabilities to handle ProcessServicesClient");
-        }
-
     }
 
-    private void startAndCompleteUsertaskProcess() {
```

Comment:
How is this change related to JUnit categories?

Reply:
Request to split testProcessFromExternalMavenRepo() into two (jBPM and Drools) from PR114.